### ISSUE TITLE
can i use it to communicate between arduino and node mcu

### ISSUE THREAD

**Issue by amrithmmh** [madsci1016/Arduino-EasyTransfer#19]
I want to add simple serial communication(send mutiple data types) between arduino and node mcu(both using arduino IDE).Is the library supported by both?

**Comment by kapil97** [comment]
Yes, you can send and receive data between node mcu and arduino by software serial library or the esp8266WiFi if using shield for arduino

**Comment by Ganesh303** [comment]
is there any special limitation of library with node mcu. because it cant sent more than 9 bytes with node mcu.
